PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -1161,6 +1162,8 @@ void PMTraceConsumer::HandleWin32kEvent(EVENT_RECORD* pEventRecord)
         {
             TRACK_PRESENT_PATH(eventIter->second);
 
+            event.SeenInFrameEvent = true;
```

Comment:
I thought the ordering here was `ScheduleCompositionSurfaceUpdate` first, and then `InFrame` afterwards. Is it inverted?

Reply:
Yeah, InFrame comes first. From test_case_1.etl
![image](https://user-images.githubusercontent.com/1754267/117917377-ca629500-b29d-11eb-8478-0e15ba2aa4f0.png)

(A second image to fit the Composition Surface Luid in)
![image](https://user-images.githubusercontent.com/1754267/117917410-db130b00-b29d-11eb-8169-4ab0ddd63334.png)

Reply:
Huh, fair enough.